src/app/components/result/result.component.ts

```diff
@@ -227,20 +227,48 @@ export class ResultComponent implements OnInit, OnDestroy {
 
         const result = `
           <!doctype html>
-          <html class="no-js" lang="${this.language}">
+          <html lang="${this.language}">
             <head>
               <meta charset="UTF-8">
-              <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
               <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
               <title>Zonemaster TEST</title>
```

Comment:
I suggest that the title is changed to "Zonemaster Test Result" and if possible and simple also add the domain name in the title, e.g. "Zonemaster Test Result iis.se"